src/create-state.js

```diff
@@ -23,7 +23,14 @@ module.exports = function( vnid, data, lm ) {
            throw new Error("Unable to create state because no vnid was provided");
        }
 
-       var stateLm = lm || createLm();
+       var stateLm;
+       if ( _.isUndefined( data ) && _.isUndefined( lm ) ) { 
+           // If both data & lm are undefined, we are initializing - accept an undefined lm
+           stateLm = lm;
```

Comment:
I would rather have the data and lm properties there, but with undefined values.  But I guess it could be clearer if written as:
<pre>
stateLm = undefined;
</pre>